packages/x-date-pickers-pro/src/MultiInputDateTimeRangeField/MultiInputDateTimeRangeField.types.ts

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
```

Comment:
Can we merge the `.interface.ts` file into this one for all multi input pickers ?
It's a mistake on my side for the existing one

Reply:
I though you had a really precise definition of what should be in type and what in the interface 😂